src/git.md

```diff
@@ -441,6 +441,44 @@ tradeoff. The main advantage is the generally linear commit history. This
 greatly simplifies bisecting and makes the history and commit log much easier
 to follow and understand.
 
+## Review suggestions
```

Comment:
To me, "Review suggestions" sounds like we're telling people to review their suggestions.

```suggestion
## Tips for reviewing
```